circle.yml

```diff
@@ -5,11 +5,9 @@ machine:
     USE_STRIPE_STANDALONE_ACCOUNTS: 'true'
 dependencies:
   pre:
+    - gem update bundler
     - sudo apt-get install libjpeg62; sudo apt-get install ghostscript
     - sudo apt-get update; sudo apt-get install libicu52
-    - curl --output /home/ubuntu/bin/phantomjs-2.0.1-linux-x86_64-dynamic https://s3.amazonaws.com/circle-support-bucket/phantomjs/phantomjs-2.0.1-linux-x86_64-dynamic
-    - chmod a+x /home/ubuntu/bin/phantomjs-2.0.1-linux-x86_64-dynamic
-    - sudo ln -s --force /home/ubuntu/bin/phantomjs-2.0.1-linux-x86_64-dynamic /usr/local/bin/phantomjs
```

Comment:
does circle work now or ?

Reply:
https://circleci.com/gh/LocalOrbit/localorbit/6216 says timeout and has some failures...

Reply:
Don't worry about Circle, it's been timing out for a long time. I'm working  on Circle 2.0